docs/csharp/programming-guide/concepts/async/index.md

```diff
@@ -36,7 +36,7 @@ Asynchrony proves especially valuable for applications that access the UI thread
  The async-based approach adds the equivalent of an automatic transmission to the list of options that you can choose from when designing asynchronous operations. That is, you get all the benefits of traditional asynchronous programming but with much less effort from the developer.  
   
 ##  <a name="BKMK_HowtoWriteanAsyncMethod"></a> Async methods are easier to write  
- The [async](../../../../csharp/language-reference/keywords/async.md) and [await](../../../../csharp/language-reference/keywords/await.md) keywords in C# are the heart of async programming. By using those two keywords, you can use resources in the .NET Framework, .NET Core, or the Windows Runtime to create an asynchronous method almost as easily as you create a synchronous method. Asynchronous methods that you define by using `async` and `await` are referred to as *async methods*.  
+ The [async](../../../../csharp/language-reference/keywords/async.md) and [await](../../../../csharp/language-reference/keywords/await.md) keywords in C# are the heart of async programming. By using those two keywords, you can use resources in the .NET Framework, .NET Core, or the Windows Runtime to create an asynchronous method almost as easily as you create a synchronous method. Asynchronous methods that you define by using `async` keyword are referred to as *async methods*.  
```

Comment:
I apologize, @pkulikov -- I missed this one earlier. There should be a "the" before `async` in the last sentence.

Reply:
@rpetrusha that's fixed now.